piksi_tools/console/tracking_view.py

```diff
@@ -186,7 +262,12 @@ def __init__(self, link):
     self.plot.legend.tools.append(LegendTool(self.plot.legend,
                                   drag_button="right"))
     self.link = link
-    self.link.add_callback(self.tracking_state_callback, SBP_MSG_TRACKING_STATE)
+    self.link.add_callback(self.tracking_state_callback,
```

Comment:
You can actually specify the messages in a list through the add_callback function:
 self.link.add_callback(self.tracking_state_callback, [SBP_MSG_TRACKING_STATE_DEP_B, SBP_MSG_TRACKING_STATE ] )